photography-site-demo.js/tests/vectorSearchTest.test.js

```diff
@@ -0,0 +1,4648 @@
+        0.015071888,
+        -0.0025820557,
+        -0.0072214114,
+        -0.038180403,
+        0.0004954691,
+        0.006727146,
+        0.0049073505,
+        -0.008922455,
+        0.009975176,
+        0.003944495,
+        -0.021953091,
+        -0.015277297,
+        0.011965076,
+        -0.013075569,
+        0.0040792953,
+        0.0052636066,
+        0.013223207,
+        0.037384447,
+        -0.007542363,
+        0.00013530116,
+        0.015611087,
+        0.020926045,
+        -0.027653191,
+        -0.008530894,
+        0.003257659,
+        -0.000086005006,
+        -0.028936999,
+        0.010796812,
+        0.01692057,
+        0.0037198293,
+        -0.020322656,
+        0.01413471,
+        0.012080619,
+        0.017729368,
+        -0.0072214114,
+        -0.011663382,
+        -0.024045695,
+        -0.016214475,
+        -0.0059215575,
+        0.012921512,
+        -0.025701806,
+        -0.019205745,
+        0.014468499,
+        0.005780339,
+        0.012793131,
+        0.012106295,
+        -0.01472526,
+        -0.015046212,
+        -0.0002336929,
+        0.013030636,
+        -0.011862372,
+        -0.035124946,
+        -0.012773874,
+        0.021978768,
+        0.01074546,
+        -0.0021583997,
+        -0.0045157894,
+        0.0015999438,
+        -0.008267714,
+        0.020656446,
+        0.0024472561,
+        -0.019128717,
+        -0.010443766,
+        0.029065378,
+        0.01778072,
+        0.013813757,
+        0.0088390075,
+        -0.0105015375,
+        0.015675277,
+        0.011535001,
+        0.00043809897,
+        -0.006069195,
+        -0.00457677,
+        -0.0011401806,
+        -0.021670654,
+        -0.003992638,
+        -0.051891457,
+        0.004165952,
+        -0.0145968795,
+        0.006483223,
+        -0.007189316,
+        0.009750511,
+        -0.023776095,
+        0.00862076,
+        0.025355179,
+        -0.00437778,
+        0.004637751,
+        -0.0062746042,
+        0.029887015,
+        -0.029835662,
+        0.008197105,
+        -0.036229018,
+        -0.04411159,
+        -0.0062200422,
+        0.003350735,
+        0.02434097,
+        -0.032146513,
+        0.002440837,
+        0.007709258,
+        -0.028423475,
+        0.017600987,
+        0.010758298,
+        0.0064575467,
+        0.026908584,
+        0.0065923464,
+        0.001299854,
+        0.007933924,
+        -0.018743575,
+        -0.032172192,
+        0.00305225,
+        0.006428661,
+        -0.0041499045,
+        -0.028038334,
+        -0.014365794,
+        0.0028356076,
+        -0.010899517,
+        -0.013954977,
+        -0.0036042868,
+        -0.0018149815,
+        0.0023381326,
+        -0.013621187,
+        0.00842177,
+        0.011830277,
+        -0.030605946,
+        0.00004252609,
+        -0.027036965,
+        -0.011419458,
+        0.014186062,
+        0.012651913,
+        -0.021709168,
+        0.0055781393,
+        -0.0070224213
```

Comment:
I would recommend either moving these vectors into separate file, like `photo2vector.json`, and importing them separately. Or using programatically generated test vectors, like `[...Array(1535).fill(0), 1]` to create a vector with 1535 0's followed by a 1. I think the programatically generated test vectors approach is better, because it makes the tests more readable.

Reply:
thanks, solved!